shell/browser/ui/cocoa/electron_ns_window.mm

```diff
@@ -19,8 +22,65 @@
 
 @interface NSWindow (PrivateAPI)
 - (NSImage*)_cornerMask;
+- (long long)_resizeDirectionForMouseLocation:(CGPoint)location;
```

Comment:
Actually, sorry, it looks like lint is mad about this typing:
```
(cpplint) Use int16/int64/etc, rather than the C type long
```

Reply:
it's very funny to me that _that_ is what the linter complains about in this change :D